plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -103,7 +106,7 @@ private KuduPageSink(
 
         this.table = table;
         this.session = clientSession.newSession();
-        this.session.setFlushMode(SessionConfiguration.FlushMode.AUTO_FLUSH_BACKGROUND);
+        this.session.setFlushMode(SessionConfiguration.FlushMode.AUTO_FLUSH_SYNC);
```

Comment:
Was this introduced in the latest version ?

Reply:
No, this config also existed in 1.10.0. 

See the PR description, the only way I could reproduce this bug (https://github.com/trinodb/trino/issues/5687) was by upgrading the kudu client to 1.15.0 which triggered timeouts to kudu. 

That is why I fixed this bug in this upgrade client PR.